lib/modules/manager/pub/artifacts.ts

```diff
@@ -60,16 +76,15 @@ export async function updateArtifacts({
           constraint,
         },
       ],
-    });
+    };
 
+    await exec(cmd, execOptions);
     const newLockFileContent = await readLocalFile(lockFileName, 'utf8');
-    if (
-      oldLockFileContent === newLockFileContent ||
-      newLockFileContent === undefined
```

Comment:
Doesn't seem like any other managers are checking `newLockFileContent === undefined` so I removed it